libmproxy/flow.py

```diff
@@ -614,6 +619,43 @@ def killall(self, master):
         self.flows.kill_all(master)
 
 
+class Plugins(object):
```

Comment:
This was a placeholder for the "action plugins" in my earlier pull request... I had both "view plugins" (now custom content views) and "action plugins" grouped together in `Plugins`. ContentViews don't need to be in that object, but registering actions & their options will need storage somewhere.